src/03-deals/deals.apib

```diff
@@ -88,11 +88,11 @@ Get a list of deals.
                 + source (object)
                     + type: `dealSource` (string)
                     + id: `52a4bbb2-d196-48d7-8e43-9ec2f44d06f9` (string)
-                + lost_reason (object)
-                    + reason (object)
+                + lost_reason (object, nullable)
+                    + reason (object, nullable)
                         + type: lostReason (string)
                         + id: `4e81f3c4-7dca-44cb-b126-6158bec392a2` (string)
-                    + remark: `Too expensive` (string)
+                    + remark: `Too expensive` (string, nullable)
```

Comment:
Q: can it happen that `lost_reason` is provided, but both or either reason or remark are `null`?

Reply:
Explained by @luisaterrabrito IRL; if the reason is unknown, there is no `lostReason` set.